### PR TITLE
Classifybutton fixes #48

### LINKED ISSUE #48: Make a button for the classify page

### PR DESCRIPTION
**Pull request by Azkle**

### CODE REVIEW

**Comment by Azkle** [comment]
To time the same commit I think something went wrong in the rebase.

File: app/views/about.html [hunk 1]
```diff
@@ -25,7 +25,7 @@
     <p>
         Want to make MoodCat a better place?
         You can help us improving Moodcat in a couple of ways.
-        All users can participate in our <a href="/rate-the-song">rating game</a>.
+        All users can participate in our <a href="#/classify">rating game</a>.
```
Comment (JeremybellEU) [inline app/views/about.html]: Should this also be a ui-sref? (as with index.html)

Comment (JWGmeligMeyling) [inline app/views/about.html]: yes.

File: app/views/menu.html [hunk 2]
```diff
@@ -1,5 +1,7 @@
 <ul class="nav navbar-nav">
   <li class="active"><a href="#/">Home</a></li>
+  <li><a ng=href="#/classify">Rating game</a></li>
```
Comment (JWGmeligMeyling) [inline app/views/menu.html]: Here as well.

File: app/views/about.html (hunk 1 above)
Comment (Azkle) [inline app/views/about.html]: Ok, I'll change it. 
The href is already in master.

File: app/views/menu.html (hunk 2 above)
Comment (Azkle) [inline app/views/menu.html]: This is how the menu was made by Gijs.
I sticked to the structure as you can see.
I just added another button, the uit-sref is in index.html.

There should be a reason why it's this way, I think. 
You should ask @GijsWeterings

**Comment by JWGmeligMeyling** [comment]
:+1:

**Comment by GijsWeterings** [comment]
LGTM

**Pull request merged**